portal/guides/on-demand-minting.mdx

```diff
@@ -25,37 +27,22 @@ Before we start make sure you have the dependencies installed and understand how
 
 After instantiating the `sdk`, we're going to:
 
-1. Build a component for the `connect wallet` button to... connect our wallet and use it to create a signature
+1. Build a component for the `connect wallet` button to connect our wallet and use it to create a signature
```

Comment:
i know it was like this already - but I don't see anywhere an explanation of `why` would I want to use this vs a regular mint or a drop. I think it's super important to explain the distinction otherwise I wouldn't really get why I would use this